misk-testing/src/main/kotlin/misk/testing/LogLevelExtension.kt

```diff
@@ -0,0 +1,46 @@
+package misk.testing
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.Logger
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.slf4j.LoggerFactory
+import javax.inject.Inject
+
+/**
+ * Helps you set the log level for misk tests. It would default all test by default to Log
```

Comment:
TODO: improve the doc 😅

Reply:
imo doc on LogLevel is more valuable since that one is external-facing.